package.json

```diff
@@ -1,31 +1,31 @@
 {
   "name": "reviews",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Is this actually an update that breaks prior compatibility, or does it simply extend functionality?

It might just be 1.0.1.

A minor quibble, to be sure. I don't think most people are paying attention to their version numbers.

Reply:
I'm versioning as Major.Minor.Patch. Since this version contains backend switch plus an extension of APIs, I chose minor over patch where patch represents bug fixes.